.github/workflows/load-tests.yml

```diff
@@ -0,0 +1,106 @@
+name: Load Tests
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+jobs:
+  determine-service-matrix:
+    runs-on: ubuntu-latest
+    outputs:
+      services: ${{ steps.determine-matrix.outputs.changes }}
+    steps:
+      - name: Check Out Repo
+        uses: actions/checkout@v4
+      - uses: dorny/paths-filter@v3
+        id: determine-matrix
+        with:
+          # Note, 'common' is for future use--when code is factored out into common libraries,
+          # but it's set up here so that the workflow logic could be tested with it present
+          filters: |
+            common:
+              - '*'
+              - '.github/**'
+            account:
+              - '*'
+              - '.github/**'
+              - 'services/account/**'
+            graph:
+              - '*'
+              - '.github/**'
+              - 'services/graph/**'
+            content-publishing:
+              - '*'
+              - '.github/**'
+              - 'services/content-publishing/**'
+            content-watcher:
+              - '*'
+              - '.github/**'
+              - 'services/content-watcher/**'
+
+  build:
+    name: '[${{ matrix.service }}] Load Test'
+    runs-on: ubuntu-latest
+    needs: determine-service-matrix
+    strategy:
+      fail-fast: false
+      matrix:
+        service: ${{ fromJson(needs.determine-service-matrix.outputs.services) }}
+        exclude:
+          - service: common
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install Node.js
```

Comment:
Why node?

Because we need to run the setup script `npm run main` in `tools/ci-k6`